zellij-tile/src/lib.rs

```diff
@@ -19,8 +22,16 @@ macro_rules! register_plugin {
         }
 
         fn main() {
+            let options = $crate::shim::object_from_stdin().map_err(|err| {
+                eprintln!(
+                    "Your plugins configuration had a derialization error: {:?}.",
```

Comment:
Just a small English nitpick, I think this should be `plugin's` here?

Also, does this print this error if a plugin is left unconfigured and the default is used? If so, we might want to change the error to make it a little more descriptive.